libass/ass_utils.h

```diff
@@ -160,10 +160,10 @@ static inline int double_to_d22(double x)
     return (int) (x * 0x400000);
 }
 
-// Calculate cache key for a rotational angle in radians
+// Calculate cache key for a rotational angle in degrees
 static inline int rot_key(double a)
```

Comment:
I missed that function in my quantization PR: its uses have got eliminated there. So it should be safe to remove it.